Add wilmersdorf, tron legacy and yay-evil distro

ianpan870102 opened the pull request:
Links to the repos:
- Wilmersdorf: https://github.com/ianpan870102/wilmersdorf-emacs-theme
- Tron Legacy: https://github.com/ianpan870102/tron-legacy-emacs-theme
- Yay-Evil Distro: https://github.com/ianpan870102/.emacs.d

ianpan870102 commented:
Wilmersdorf and Tron Legacy have both proven popularity on forums like Reddit's r/emacs and are featured on GNU Emacs Theme website.

The Yay-Evil Distro that I currently maintain contains a comprehensive starter kit experience with good defaults, documented in literate programming Org style.

agsdot commented on README.org:
```diff
@@ -836,6 +836,8 @@ For additional git related emacs packages to use or to get inspiration from, tak
    - [[https://framagit.org/steckerhalter/grandshell-theme][Grandshell-theme]] - /(dark)/ *[256color]* Theme with intensive colors.
    - [[https://github.com/kunalb/poet][Poet]] - /(light/dark)/ A lightweight theme that plays well with prose and org-mode.
    - [[https://github.com/ogdenwebb/emacs-kaolin-themes][Kaolin-themes]] - (theme package) Set of eye pleasing themes for GNU Emacs. Supports both GUI and terminal.
+   - [[https://github.com/ianpan870102/wilmersdorf-emacs-theme][Wilmersdorf-theme]] - /(dark)/ Emacs theme with dark subtle syntax highlighting
```
Please end the description with 

syntax highlighting.

ianpan870102 replied:
Fixed! Please refer to my latest comment.

agsdot commented on README.org:
```diff
@@ -836,6 +836,8 @@ For additional git related emacs packages to use or to get inspiration from, tak
    - [[https://github.com/kunalb/poet][Poet]] - /(light/dark)/ A lightweight theme that plays well with prose and org-mode.
    - [[https://github.com/ogdenwebb/emacs-kaolin-themes][Kaolin-themes]] - (theme package) Set of eye pleasing themes for GNU Emacs. Supports both GUI and terminal.
+   - [[https://github.com/ianpan870102/wilmersdorf-emacs-theme][Wilmersdorf-theme]] - /(dark)/ Emacs theme with dark subtle syntax highlighting
+   - [[https://github.com/ianpan870102/tron-legacy-emacs-theme][Tron-Legacy-Theme]] - /(dark)/ Custom theme inspired by Tron: Legacy
```
Once again. Add a period to finish the description.

Tron: Legacy.

ianpan870102 replied:
Fixed! Please refer to my latest comment.

ianpan870102 commented:
Thanks for your time, @agsdot. I've fixed the typo and squashed my latest commit to the previous one to keep the commit history clean. Let me know if it's all good!

![Screenshot 2019-07-26 at 10 06 04 AM](https://user-images.githubusercontent.com/36686900/61920682-0293c900-af8d-11e9-92f1-2056d49c80cb.png)

agsdot commented:
Thanks @ianpan870102!
